ui/pages/onboarding-flow/creation-successful/creation-successful.js

```diff
@@ -32,43 +32,45 @@ export default function CreationSuccessful() {
 
   return (
     <div className="creation-successful" data-testid="creation-successful">
-      <Box textAlign={TEXT_ALIGN.CENTER}>
+      <Box textAlign={TextAlign.Center}>
         <img src="./images/tada.png" />
-        <Typography
-          variant={TypographyVariant.H2}
-          fontWeight={FONT_WEIGHT.BOLD}
+        <Text
+          variant={TextVariant.headingLg}
+          as="h2"
+          fontWeight={FontWeight.Bold}
           margin={6}
         >
           {t('walletCreationSuccessTitle')}
-        </Typography>
-        <Typography variant={TypographyVariant.H4}>
+        </Text>
+        <Text variant={TextVariant.headingSm} as="h4">
           {t('walletCreationSuccessDetail')}
-        </Typography>
+        </Text>
       </Box>
-      <Typography
-        variant={TypographyVariant.H4}
-        boxProps={{ align: AlignItems.flexStart }}
+      <Text
+        variant={TextVariant.headingSm}
+        as="h4"
+        align={AlignItems.flexStart}
```

Comment:
I think the boxProps `align` prop was a mistake. It should be `alignItems` and even then I'm not sure why you would use it here

```suggestion
```